Add Packer File for setup-server.yml AMI

jasonrogena opened the pull request:
Add a Packer file for creating an AMI from the setup-server.yml
playbook. Also add instructions on how to run the Packer file.

Signed-off-by: Jason Rogena <jason@rogena.me>

jasonrogena committed:
Add Packer file for setup-server.yml AMI

Add a Packer file for creating an AMI from the setup-server.yml
playbook. Also add instructions on how to run the Packer file.

Signed-off-by: Jason Rogena <jason@rogena.me>

morrismukiri reviewed (commented):
Might be good to include a sample var file. Otherwise LGTM

morrismukiri reviewed (commented):
We could also define the user variables in the template to set the defaults and to self-document.
REF: https://www.packer.io/docs/templates/user-variables.html#usage

jasonrogena committed:
Add default variables for server-setup Packer

Add the default variables for the Packer server-setup AMI file, as
suggested by Morris Mukiri.

Signed-off-by: Jason Rogena <jason@rogena.me>

jasonrogena committed:
Add server-setup Packer file to ZEIR sample

Add a sample variant of the ami-server-setup Packer file in the
zeir-sample/staging inventory.

Signed-off-by: Jason Rogena <jason@rogena.me>
